server/util.go

```diff
@@ -41,6 +42,9 @@ import (
 const (
 	requestTimeout  = etcdutil.DefaultRequestTimeout
 	slowRequestTime = etcdutil.DefaultSlowRequestTime
+
+	// PrivateDirMode grants owner to make/remove files inside the directory.
+	PrivateDirMode = 0700
```

Comment:
I prefer 0755 here. Sometimes we may want to see the log with another account, especially for docker.